dotnet/samples/Demos/VectorStoreRAG/README.md

```diff
@@ -55,6 +59,7 @@ For Azure OpenAI Embeddings, you need to add the following secrets:
 
 ```cli
 dotnet user-secrets set "AIServices:AzureOpenAIEmbeddings:Endpoint" "https://<yourservice>.openai.azure.com"
+dotnet user-secrets set "AIServices:AzureOpenAIEmbeddings:DeploymentName" "text-embedding-ada-002"
```

Comment:
Wonder if we should have OpenAI instead of Azure for the demonstration.

Reply:
At the moment I cannot get open ai working for me, so that's why I'm using azure open ai.  Once I can get open ai working, I'd be happy to switch.